ArchiSteamFarm/WebBrowser.cs

```diff
@@ -643,31 +686,31 @@ public sealed class WebBrowser : IDisposable {
 		}
 
 		[ItemCanBeNull]
-		private async Task<StringResponse> UrlPostToString(string request, IReadOnlyCollection<KeyValuePair<string, string>> data = null, string referer = null, ERequestOptions requestOptions = ERequestOptions.None, byte maxTries = MaxTries) {
+		private async Task<StreamResponse> UrlPostToStream(string request, IReadOnlyCollection<KeyValuePair<string, string>> data = null, string referer = null, ERequestOptions requestOptions = ERequestOptions.None, byte maxTries = MaxTries) {
 			if (string.IsNullOrEmpty(request) || (maxTries == 0)) {
 				ArchiLogger.LogNullError(nameof(request) + " || " + nameof(maxTries));
 
 				return null;
 			}
 
-			StringResponse result = null;
+			StreamResponse result = null;
 
 			for (byte i = 0; i < maxTries; i++) {
-				using HttpResponseMessage response = await InternalPost(request, data, referer).ConfigureAwait(false);
+				HttpResponseMessage response = await InternalPost(request, data, referer).ConfigureAwait(false);
```

Comment:
Lack of `using` here has a potential to cause delayed TCP sockets cleanup, which might destabilize the networking, are you sure it's needed?

Reply:
Yes, because using `using` will close underlying response stream, which is unacceptable (it should be closed by the consumer of the stream).

Reply:
I wonder if there is any way to ensure it's closed on our side then, would it make sense to make `StreamResponse` `IDisposable` and use it for duration of stream parsing?

Reply:
`HttpResponseMessage` only disposes `HttpContent _content` under the hood, which disposes `Stream _contentReadStream`, so I am not sure if that needed at all, because consumer anyway closes the stream.

Reply:
We should not rely on current implementation of `Dispose()` in classes we're using, especially since `StreamResponse` is `internal`, and we can easily dispose it after stream is reading, by just `(using streamResponse = new StreamResponse(...)) {`